python/tvm/topi/random/kernel.py

```diff
@@ -466,3 +466,54 @@ def gen_ir(out_ptr):
     out_ary = tvm.nd.array(np.ones((1,), "uint64"), device)
     tvm.build(s, [f], target=target)(out_ary)
     return out_ary.asnumpy()[0] == 0
+
+
+def uniform(gen, low, high, out_shape, out_dtype):
+    """Draw samples from a uniform distribution.
+
+    Samples are uniformly distributed over the half-open interval [low, high)
+    (includes low, but excludes high). In other words, any value within the
+    given interval is equally likely to be drawn by uniform.
+
+    Parameters
+    ----------
+    gen : Tensor[10, uint64]
```

Comment:
What is the meaning of `10`?

Reply:
This is the `ThreefryKeyType` introduced in #7083. Please refer to: https://github.com/apache/tvm/blob/c999a840cb5579c493f5b5e7f20bc619260dad08/src/relay/op/random/kernel.cc#L28

Reply:
If so, let us add comment describe what is the meaning of 10.

Reply:
You could probably say `ThreefryKey` instead of `Tensor[10, uint64]`

Reply:
Fixed.